io.embrace.sdk/Scripts/EmbraceApi.cs

```diff
@@ -68,8 +69,9 @@ internal interface IEmbraceApi: ILogsApi, IMomentsApi, INetworkRequestApi, ISess
         /// Starts instrumentation of the application using the Embrace SDK. This should be called during creation of the application, as early as possible.
         /// See Embrace Docs for integration instructions. For compatibility with other SDKs, the Embrace SDK must be initialized after any other SDK.
         /// </summary>
+        /// <param name="args">Startup arguments to configure the SDK. REQUIRED on iOS</param>
         /// <param name="enableIntegrationTesting">If true, debug sessions (those which are not part of a release APK) will go to the live integration testing tab of the dashboard. If false, they will appear in 'recent sessions'.</param>
-        void StartSDK(bool enableIntegrationTesting = false);
+        void StartSDK(EmbraceStartupArgs? args = null, bool enableIntegrationTesting = false);
```

Comment:
It seems like `enableIntegrationTesting` should be included in `args`. If we want to keep it at all that is, we've deleted it on Android

Reply:
If we've deleted it on Android, I know that iOS doesn't use it at all. As a result, I think we can just remove it.

Reply:
Deleting this would directly force us to move to a 2.0 as that would be a breaking change for all customers regardless.

Reply:
Something to discuss with Eliab in that case - but yeah, we can definitely rip this out whenever the next major version is